src/server/game/AuctionHouseBot/AuctionHouseBot.cpp

```diff
@@ -111,6 +142,8 @@ void AuctionBotConfig::SetConfig(AuctionBotConfigFloatValues index, char const*
 //Get AuctionHousebot configuration file
 void AuctionBotConfig::GetConfigFromFile()
 {
+	SetConfig(CONFIG_AHBOT_ACCOUNT_ID, "AuctionHouseBot.Account", 0);
```

Comment:
Replace `<Tab>` with 4 spaces. GitHub displays `<Tab>` as 8 spaces and TC will not accept Tabs in the code. :)